Modules/main.c

```diff
@@ -435,7 +435,9 @@ pymain_clear_pymain(_PyMain *pymain)
 {
 #define CLEAR(ATTR) \
     do { \
-        PyMem_RawFree(ATTR); \
+        if (ATTR != NULL) { \
```

Comment:
The if is useless: PyMem_RawFree(NULL) does nothing, the call is safe.

Reply:
Ok, removed